### PR TITLE
Get rid of default config, sort options in template by topic and importance

### PR DESCRIPTION
**Pull request by mlange-42**

### CODE REVIEW

File: src/parser/md.rs [hunk 1]
```diff
@@ -137,10 +137,10 @@ impl Default for MdParser {
             fence_sequence: String::from("```"),
             fence_sequence_alt: String::from("~~~"),
             comments_as_aside: false,
-            comment_start: String::from("//"),
+            comment_start: String::from("//-"),
             interpolation_start: String::from("@{"),
             interpolation_end: String::from("}"),
-            macro_start: String::from("==> "),
+            macro_start: String::from("// ==>"),
```
Comment (adamreichold) [inline src/parser/md.rs:143]: Do want to keep the defaults in the config, why do we duplicate them in the code at all?

I am also a bit confused by the default config loading in the application. Is it actually possible to successfully build project without a config resp. with the default config? Shouldn't we just always load the (defaulting via Clap config) config file path and drop the defaults here?

Comment (mlange-42) [reply src/parser/md.rs:143]: Indeed, that would be possible. It is probably from before using the template, which was introduced to have comments in the generated config.

Comment (mlange-42) [reply src/parser/md.rs:143]: I guess I can `unwrap` this: `matches.value_of("config")` as a default value is given?

Comment (mlange-42) [reply src/parser/md.rs]: ... as in the current version...

**Review by adamreichold — APPROVED** [review]

**Pull request merged**